### PR TITLE
Bug 1068980 - Moved the fullscreen back button above the image frames ta...

### PR DESCRIPTION
**Pull request by eeejay**
...g.

### CODE REVIEW

File: apps/gallery/index.html [hunk 1]
```diff
@@ -98,6 +98,7 @@ <h1 data-l10n-id="gallery" id="fullscreen-title">Gallery</h1>
           <span data-icon="delete"></span>
         </a>
       </gaia-header>
+      <button id="fullscreen-back-button-tiny" type="reset" class="tiny-only" data-icon="back"></button>
```
Comment (punamdahiya) [inline apps/gallery/index.html]: Tested and moving back button on top of frames div hides it behind the image or video poster image in full screen view.

Comment (punamdahiya) [inline apps/gallery/index.html]: Do we need to set data-l10n-id here for localization?

Comment (eeejay) [inline apps/gallery/index.html]: Good catch! I'll add a label, and raise the z-index.

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6078097765e5)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bfe2ee2e23f8)

**Pull request merged**